djangocms_transfer/serializers/django_cms.py

```diff
@@ -0,0 +1,77 @@
+import base64
+import io
+
+from django.core.files import File as DjangoFile
+from django.core.serializers import base
+from django.core.serializers.python import (
+    Serializer as PythonSerializer,
+    Deserializer as PythonDeserializer,
+    _get_model,
+)
+
+try:
+    from filer.fields.image import FilerImageField
+    from filer.models import Image
+
+    has_filer = True
+except ImportError:
+    has_filer = False
+
+
+class FilerImageFieldSerializer:
+    @classmethod
+    def serialize(cls, field_instance):
+        serializer = Serializer()
+        _image_plugin_data = serializer.serialize((field_instance,))[0]
+        _file_plugin_data = serializer.serialize(
+            (field_instance.file_ptr,), fields=["original_filename", "mime_type"]
+        )[0]
+        base64_image = base64.b64encode(field_instance.file.read())
+
+        _plugin_data = _image_plugin_data["fields"]
+        _plugin_data.update(_file_plugin_data["fields"])
+        _plugin_data["file_content"] = base64_image.decode()
+        return _plugin_data
+
+    @classmethod
+    def deserialize(cls, serialized_data):
+        base64_image = base64.b64decode(serialized_data.pop("file_content"))
+
+        filename = serialized_data["original_filename"]
+        file_obj = DjangoFile(io.BytesIO(base64_image), name=filename)
+        image = Image.objects.create(
+            **serialized_data,
+            file=file_obj,
+        )
+
+        return image.pk
+
+
+class Serializer(PythonSerializer):
+    def handle_fk_field(self, obj, field):
+        if has_filer and isinstance(field, FilerImageField):
+            field_instance = getattr(obj, field.name)
+            self._current[field.name] = FilerImageFieldSerializer.serialize(
+                field_instance
+            )
+        else:
+            super(Serializer, self).handle_fk_field(obj, field)
+
+
+def Deserializer(object_list, **options):
```

Comment:
Pep8: this should be lowercase if it's a function, however, it appears that you might have meant this to be a class with a deserialize method??? It seems by the name that is should be a method with maybe a static method. Anyway, just a nitpick

Reply:
Hi, I can change it to lowercase. It was inspired by Django python serializer itself https://github.com/django/django/blob/main/django/core/serializers/python.py#L88

Reply:
Actually, it needs to be uppercase https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/django/core/serializers/__init__.py#L125

Reply:
@robert-foreflight what do you think about that ?